src/makeRobot.js

```diff
@@ -39,6 +40,72 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const mateRobot = {
+    name: name,
+    'chip version': version,
```

Comment:
у объекта не может быть свойств через пробел
```suggestion
    chipVersion: version,
```